projects/veda/constants.js

```diff
@@ -0,0 +1,25 @@
+const ADDRESSES = require("../helper/coreAssets.json");
+
+const legacyVaults = [
+  {
+    id: "0xeA1A6307D9b18F8d1cbf1c3Dd6aad8416C06a221",
```

Comment:
this is ether.fi liquid vault right? why are you counting it as your tvl?

Reply:
The Veda protocol is the infrastructure that is powering EtherFi Liquid

Reply:
can you explain bit more, how it works? who deployed/controls the contracts? we can count the tvl as part of only one dapp, not both

Reply:
veda isn't a dapp, it's the smart contract infrastructure that the etherfi liquid product is built on. The general idea of veda is a protocol that enables the creation of tokenized yield products. Liquid is the first instance of a tokenized yield product built on veda but there are more to come

Reply:
yes, but what does smart contract infrastructure mean? 

Is any of the vaults in this PR deployed/controlled by your team?

Reply:
The smart contracts were deployed by our team however the question of “control” is a more involved one.   High level Veda is a system that allows protocols to offer yield products which are optimized by off-chain strategists. The Veda smart contracts constraint the role of the strategist along with potential other service providers through a set of on-chain and off-chain checks and balances. The smart contracts have various roles that the protocol integrator (EtherFi in this case) can use to add security or custom controls, for example changing out the strategist.

In the case of EtherFi Liquid, the strategist is Seven Seas and EtherFi itself has an administration role in the contracts. The security model, roles, strategist rebalancing process, etc. are part of the Veda protocol.

An existing analogy for something that’s already integrated in DeFiLlama is the [Swell Vault](https://defillama.com/protocol/swell-vault#information) and [DIVA Vault](https://defillama.com/protocol/diva#information) both built by Enzyme. Both Swell and DIVA are independently reporting these vaults in their TVL, and the TVL from both vaults are also included in [Enzyme’s total TVL](https://defillama.com/protocol/enzyme-finance#information).

Reply:
Hi @g1nt0ki just following up. Let me know if this makes sense

Reply:
Hey @g1nt0ki just wanted to check in here again

Reply:
sorry, for the delay, we are still discussing how to handle this

Reply:
the issue remains that we cant count the tvl in same contract as part of two different projects, can you check with etherfi? we can maybe delist them and list this